srcpkgs/cli-visualizer/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'cli-visualizer'
+pkgname=cli-visualizer
+version=1.7
+revision=1
+build_style=gnu-configure
+build_options="pulseaudio"
+build_options_default="pulseaudio"
+make_install_args="PREFIX=${DESTDIR}/usr/bin/"
+hostmakedepends="clang"
+makedepends="fftw-devel ncurses-devel
+ $(vopt_if pulseaudio 'pulseaudio-devel')"
+depends="libfftw ncurses"
+short_desc="CLI based audio visualizer"
+maintainer="Jasper Chan <jasperchan515@gmail.com>"
+license="MIT"
+homepage="https://github.com/dpayne/cli-visualizer"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=0445fbfad9707668a70ff146f3f24308f28fa0ae67a7010bfdea0dacba75b75b
+_sconf_dir="${DESTDIR}/usr/share/examples/${pkgname}"
+
+if [[ $XBPS_MACHINE == "arm6"* ]];  then _set_arm; fi
+if [[ $XBPS_MACHINE == "armv6"* ]]; then _set_arm; fi
+if [[ $XBPS_MACHINE == "armv7"* ]]; then _set_arm; fi
+
+pre_install() {
+	mkdir -p "${DESTDIR}/usr/bin/"
+}
+
+post_install() {
+	# Files listed in install.sh
+	vsconf examples/config
+	vsconf examples/rainbow
```

Comment:
why put them in examples just to then `mv` later to _sconf_dir ? just mv from examples/rainbow directly to _sconf_dir